services/web/project/app/models.py

```diff
@@ -38,3 +38,32 @@ def password(self, plain_text_password):
 
     def check_password_correction(self, attempted_password):
         return check_password_hash(self.password, attempted_password)
+
+class Question(models.BaseModel):
+    __tablename__ = "question"
```

Comment:
should be multiple `questions`

Reply:
Resolved issue with commit: cfc52c19c713fc2c0e5ac6adf2539b3c34e00049